projects/canopy/src/lib/side-nav/side-nav-bar-item-content/side-nav-bar-item-content.component.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
More a question for everyone, but if this is all the template is, is it better to put directly into the component definition as `template` instead of using a `templateUrl` ?

Reply:
Good question, happy to move that to a discussion/issue. It might be out of scope for this PR, but could help us move faster in the future.

Reply:
Yeah, I will raise it